wiki/3_wiki.md

```diff
@@ -133,43 +117,188 @@ The technologies we use are:
+
+Once an account was created the user should change his password. He will also be able to change his username.
+
+
+#### 3.1.9 Download files
+
+This is one of the most important use cases of the application. You will be able to download files you are allowed to see with your browser.
+
+
+
+#### 3.1.10 Upload files
+
+The same way you can download files you will also be able to upload files in places you are allowed to. This way you can also overwrite files.
+
+#### 3.1.11 Search for files
+
+You will be able to search for files with a specific name within all the files you are allowed to view.
 
 
 ### 3.2 Usability
 
+We want our application to be easy to set up and easy to use. The user interface should be intuitive and self-explanatory. It should be close to different filesystem browsers you can find on operation systems.
+Setting up the application should be easy for everyone who has some experience with computers and servers. The installation will be done by a single shell script and require docker.
+
+
+
 
 ### 3.3 Reliability
 
+The provided software should be as stable as possible, so the users don't encounter problems. During the development process we will always provide two versions, one should be stable and one that already includes the latest features.
+
+
+
 #### 3.3.1 Availability
 
-### 3.4 Perfomance
+Each user will be responsible for the availability of their own instance, but we should make sure there is always a stable version available for them to install.  
+
+#### 3.3.2 Data loss
+
+We shall provide options to back up all the data stored by our application. 
+
+
+### 3.4 Performance
+
+Performance is important for all parts of our software and will always be considered during the development process.
+
+
+#### 3.4.1 App performance
+
+By using react we can provide a single-page application, that will make using the website faster than regular websites even with slightly slower internet speeds.
+
+#### 3.4.2 Storage
+
+As computer storage has become very cheap we will rather prefer an easy installation process than low storage usage. With docker the whole application will only have a single dependency, but will need more storage.
+
+#### 3.4.3 Scalability
+
+Our application will focus on private usage with a smaller set of users. By using docker we will still be able to scale some of our services to provide better performance if needed.
+
 
 ### 3.5 Supportability
 
+#### 3.5.1 Coding Standards
+WIP
+We are going to write the code by using common clean code standards. For example, we will name our variables and methods by their functionalities. This will keep the code easy to read by everyone and make further development much easier.
+
+#### 3.5.2 Testing Strategy
+WIP
+The application will have a high test coverage and all important functionalities and edge cases should be tested. Further, mistakes in the implementation will be discovered instantly, and it will be easy to locate the error.
+
```

Comment:
Better?
`The testable parts of our application e.g. RESTApi will have a high.....`